backend/include/RadioHelper.hpp

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include "Shared.hpp"
+#include "sdk.hpp"
+#include <quill/Quill.h>
+
+class RadioState {
+public:
+    double frequency;
+    bool headset;
+    bool rx;
+    bool tx;
+    bool xc;
+    bool xca;
+};
+
+class RadioHelper {
+public:
+    /**
+     * @brief Set the state of a radio.
+     *
+     * @param newState The state to set
+     * @return true The state was set successfully
+     * @return false The state was not set successfully
+     */
+    inline static bool SetRadioState(SDK* mApiServer, const RadioState& newState)
```

Comment:
Refactored code from `main.cpp` to centralize all the logic for setting radio states. This method gets used in all the places that used to directly call `SetRx`/`SetTx`/etc.

Reply:
I don't like that the SDK gets passed in to this method. I wanted to use a shared global thing like `mApiServer` but couldn't figure out where to put it or initialize it so I went this way for now.